scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/task/SchedulerVars.java

```diff
@@ -41,5 +41,7 @@
     PA_TASK_REPLICATION,
     PA_TASK_SUCCESS,
     PA_USER,
-    PA_SCHEDULER_REST_URL
+    PA_SCHEDULER_REST_URL,
+    PA_PROPAGATED_VARIABLE_PREFIX,
+    PA_TASK_VARIABLE_PREFIX
```

Comment:
The name of `PA_TASK_VARIABLE_PREFIX` should reflect that the variable is going to be discarded after the task finishes if this is the wanted behavior.